esp_bsp_generic/Kconfig

```diff
@@ -0,0 +1,467 @@
+menu "Board Support Package"
+
+    orsource "$IDF_PATH/examples/common_components/env_caps/$IDF_TARGET/Kconfig.env_caps"
+
+    config BSP_ERROR_CHECK
+        bool "Enable error check in BSP"
+        default y
+        help
+            Error check assert the application before returning the error code.
+       
+    choice BSP_PREDEFINED_BOARD
+        prompt "Predefined board"
+        default BSP_PREDEFINED_BOARD_CUSTOM
+        help
+            Predefined board set all pins, which are available on boards for buttons, LEDs, SD card and more...
+
+        config BSP_PREDEFINED_BOARD_ESP32_DEVKITC_V4
+            bool "ESP32-DevKitC V4"
```

Comment:
If I change the board in menuconfig, the pinout will not be automatically updated, right?

Reply:
It will update only first time. After sdkconfig was generated, There is no way, how to change it.